docker-entrypoint.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+
+#  Try to check if the first argument indicates that we need to run
+# elasticsearch.
+if [[ $# -lt 1 ]] || [[ "$1" == "-"* ]] || [[ "$1" == "elasticsearch" ]]; then
```

Comment:
How about separating this out as is done in the `mongo` entrypoint instead?

cf. https://github.com/docker-library/mongo/blob/master/3.0/docker-entrypoint.sh